web/angular-wallet/src/app/main/send-burst/send-burst-form/send-burst-form.component.ts

```diff
@@ -80,17 +80,25 @@ export class SendBurstFormComponent extends UnsubscribeOnDestroy implements Afte
   ngAfterViewInit(): void {
     if (this.route.snapshot.queryParams) {
       setTimeout(() => {
-        this.onRecipientChange(new Recipient(this.route.snapshot.queryParams.recipient));
-        this.fee = this.route.snapshot.queryParams.fee;
-        this.amount = this.route.snapshot.queryParams.amount;
+        this.onRecipientChange(new Recipient(this.route.snapshot.queryParams.receiver));
```

Comment:
this func is pretty huge already...I tend/recommend to extract it as private function